src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/BinaryFormatUtilitiesTests.cs

```diff
@@ -25,33 +26,58 @@ private void WriteObjectToStream(object value, bool restrictSerialization = fals
     private object? ReadObjectFromStream()
     {
         _stream.Position = 0;
-        return Utilities.ReadObjectFromStream<object>(_stream, resolver: null, legacyMode: true);
+        DataRequest request = new("test")
+        {
+            UntypedRequest = true
+        };
+
+        return Utilities.ReadObjectFromStream<object>(_stream, in request);
     }
 
     private object? ReadRestrictedObjectFromStream()
     {
         _stream.Position = 0;
-        return Utilities.ReadRestrictedObjectFromStream<object>(_stream, resolver: null, legacyMode: true);
+        DataRequest request = new("test")
```

Comment:
```suggestion
        DataRequest request = new("TEXT")
```
to indicate that this is a restricted format even though the field is not used.

Reply:
"test" is pretty standard, or perhaps "unused". I don't see that "TEXT" adds anything here for this internals unit test.

Reply:
I'm fine with "unused".